Tutorials/creating-2d-metroidvania-game-with-splashkit/2-1-drawing-options-functions-using-bitmap.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Drawing Options functions using DrawBitmap"
+description: Learn how to use drawing options functions in Splashkit and combine them to create dynamic and visually stunning graphics.
+author: Sharvani Kandala
+date: 05/05/2024
+tags: programming, graphics, Splashkit, tutorials
+sidebar: 
+   label: " -2-1 Drawing Options Functions using Bitmap"
+---
+
+## Window Setup
+
+To begin utilizing drawing options functions in Splashkit, set up the game window as follows:
+
+1. Import the necessary Splashkit headers at the beginning of your code:
+
+   ```cpp
+   #include "splashkit.h"
+   ```
+1. In the `main` function, configure the game window and any necessary resources:
+
+   ```cpp
+    int main()
+    {
+       open_window("Advanced Drawing", 800, 600);
+       load_resource_bundle("game_resources", "r");
+       return 0;
+    }
+    ```
+## Drawing Options Functions
+
+## Example 1: Applying Option Part Bmp:
+
+In this example, we'll demonstrate how to use 'Option Part Bmp' to draw a section of a bitmap, effectively cropping the image.
+
+# Function Syntax:
```

Comment:
- Make sure each heading is unique.
- Make sure that you are using the correct heading size - ###
- Change the things mentioned above in the rest of the file as well.

And not sure if the content below should go somewhere else (above this Option Part Bmp section), as it is not explaining the relevant function.